unittests/PhasarLLVM/ControlFlow/LLVMBasedICFGExportTest.cpp

```diff
@@ -0,0 +1,283 @@
+#include "gtest/gtest.h"
+
+#include <algorithm>
+#include <fstream>
+#include <string>
+#include <vector>
+
+#include "llvm/ADT/DenseMap.h"
+#include "llvm/ADT/SmallPtrSet.h"
+#include "llvm/ADT/Twine.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/Support/Casting.h"
+#include "llvm/Support/raw_ostream.h"
+
+#include "nlohmann/json.hpp"
+
+#include "phasar/Config/Configuration.h"
+#include "phasar/DB/ProjectIRDB.h"
+#include "phasar/PhasarLLVM/ControlFlow/LLVMBasedCFG.h"
+#include "phasar/PhasarLLVM/ControlFlow/LLVMBasedICFG.h"
+#include "phasar/PhasarLLVM/Passes/ValueAnnotationPass.h"
+#include "phasar/PhasarLLVM/Pointer/LLVMPointsToInfo.h"
+#include "phasar/PhasarLLVM/TypeHierarchy/LLVMTypeHierarchy.h"
+#include "phasar/Utils/LLVMIRToSrc.h"
+#include "phasar/Utils/LLVMShorthands.h"
+#include "phasar/Utils/Logger.h"
+
+#include "TestConfig.h"
+
+using namespace std;
+using namespace psr;
+
+using nlohmann::json;
```

Comment:
We recently started to completely avoid `using namespace` and try not to introduce new ones.